inference-engine/src/transformations/include/transformations/convert_reduce_to_pooling.hpp

```diff
@@ -71,6 +75,8 @@ class ngraph::pass::ConvertReduceSumToPooling: public ConvertReduceBase {
 template <class T>
 ngraph::matcher_pass_callback ConvertReduceBase::convert_reduce_to_pooling() {
     return [&](ngraph::pattern::Matcher& m) {
+        IttTask task;
```

Comment:
Is it a third way to cover matcher pass?

Reply:
Recently, the itt.hpp header library was moved from the includes directory to the src directory. Thus, this private helper class is required to use ITT counters without exposing the counters interface in transformations library interface.

Reply:
All counters was removed from matchers